docs/api/ShallowWrapper/rightContains.md

```diff
@@ -0,0 +1,64 @@
+# `.rightContains(nodeOrNodes) => Boolean`
+
+Returns whether or not the current wrapper has a node anywhere in it's render tree that looks like
+the one passed in. Equality is based on the expected element (`nodeOrNodes`) and not on the wrapper. It will determine if an element in the wrapper `looks like` the expected element by checking if all props of the expected element are present on the wrappers element and equals to each other.
+
+
+#### Arguments
+
+1. `nodeOrNodes` (`ReactElement|Array<ReactElement>`): The node or array of nodes whose presence you are detecting in the current instance's
```

Comment:
can we be very very explicit about whether this has `some` or `every` semantics?

I almost would prefer three separate methods: `containsMatchingElement`, `containsAllMatchingElements`, and `containsAnyMatchingElements` - taking a node, an array, and an array, respectively - rather than risking any confusion about what conditions cause `true` or `false` to be returned. Thoughts?